store/entity-behavior.js

```diff
@@ -83,7 +90,7 @@ D2L.PolymerBehaviors.Siren.EntityBehavior = {
 				}
 
 				this.removeListener = removeListener;
-				window.D2L.Siren.EntityStore.fetch(href, token);
+				window.D2L.Siren.EntityStore.fetch(href, token, this.disableEntityCache);
```

Comment:
do we want to revert back to default automatically so that the bypass isn't unintentionally left on?

Reply:
Is the idea that we change this via an attribute? In which case flipping it back might not be the best idea.

Reply:
Not sure I'm following... yeah what I was imagining was passing this in as an attribute, otherwise the `fetch` might happen before the web component can switch it from `false` to `true` (I was seeing that setting it in `ready` or `attached` might be too late.  So like:
`<component href="" token="" disableEntityCache>`, and then the consumer could flip it on and off at points throughout the code if it made sense.

Reply:
My only concern is leaving the attribute on means you are updating other entity observers on the page.  Which could get nasty real quick until we make the notifier smarter.

This is a good tool to have in our pockets, just gotta make sure we watch it.